.circleci/config.yml

```diff
@@ -0,0 +1,142 @@
+version: 2.1
+orbs:
+  slack: circleci/slack@4.1.3
+jobs:
+  terraform_lint:
+    docker:
+      - image: quay.io/astronomer/ci-terraform:2020-12
+    steps:
+      - checkout
+      - run: cp providers.tf.example providers.tf
```

Comment:
could you potentially put all that into a bash script and just call it?  might be easier to edit, test, and understand.

thoughts?

Reply:
I definitely like the pattern of writing scripts and calling those vs embedding shell scripts in yaml. It is difficult to spot bugs in yaml embedded shell scripts. IE: we can't `shellcheck` yaml embedded shell scripts.

Reply:
@tommyk @danielhoherd Same thing we're doing with the `pipeline/run_terraform.sh` script in `terraform_apply` and `terraform_destroy` jobs? Should we use the same script or create a new one for these linting steps?

Reply:
I would say yes, just like that. Whether or not it's the same script isn't the important bit, so just do what makes sense. The important thing is that doing it as a script makes it cleaner, lintable, and portable.